internal/testutils/dummy_identity.go

```diff
@@ -0,0 +1,45 @@
+package testutils
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/redhatinsights/platform-go-middlewares/identity"
+)
+
+// accountNumber to be used in the tests.
+const accountNumber = "12345"
+
+// orgId to be used in the tests.
+const orgId = "abc-org-id"
```

Comment:
To be consistent with reality, orgId is also a decimal number.

If something is weak in mocking, than it's the test data - very often out of reality.

Reply:
are you sure? I've copied it from other testing data, but it is entirely possible those data were wrong, so just asking if we have a correct understanding of what orgID should look like ^_^

Reply:
I am not asking to mix code, I am asking for a better name for the package. it can live in internal/test/xxx.

the WithIdentity function I think might be promoted to the production code, I find it useful, we have the same in Foreman as "[User.as](http://user.as/)_admin"